.github/workflows/test.yml

```diff
@@ -13,6 +13,9 @@ jobs:
   cypress:
     name: Run cypress tests
     runs-on: ubuntu-latest
+    container: 
+      image: cypress/browsers:node16.17.1-chrome106-ff105-edge
+      options: --user 1001
```

Comment:
What is --user 1001 indicates?

Reply:
There was a permissions issue with the docker container. [This discussion here](https://github.com/cypress-io/github-action/issues/446#issuecomment-987015822) outlines the issue and temporary fix.